lib/matplotlib/_mathtext.py

```diff
@@ -473,57 +473,37 @@ def _get_glyph(self, fontname: str, font_class: str,
     # The Bakoma fonts contain many pre-sized alternatives for the
     # delimiters.  The AutoSizedChar class will use these alternatives
     # and select the best (closest sized) glyph.
+    _latex_sizes = ('big', 'Big', 'bigg', 'Bigg')
```

Comment:
minor typo two lines above, while you're at it: it's AutoHeightChar, not AutoSizedChar.